support/Environments/wheeler_clang.sh

```diff
@@ -59,13 +59,18 @@ spectre_run_cmake() {
         return 1
     fi
     spectre_load_modules
+    # Notes:
```

Comment:
This might be the opposite of what we want for HDF5 support, though... 

@knelli2 I think you were having issues with conflicting HDF5 libraries being used. Does using only the Anaconda-supplied one work? Or is using the more recent 1.12 better?

Reply:
You can always add `HDF5_ROOT` here to select a specific HDF5 if needed.

Reply:
On wheeler, using the Anaconda-supplied HDF5 works fine. However, even if you have other system HDF5 modules loaded, cmake will only find the Anaconda-supplied version. @nilsleiffischer's solution of just adding `HDF5_ROOT` works and I've used it.